conf/messages.en

```diff
@@ -518,10 +518,14 @@ cds.registration.you-need-a-different-service-iom.info=To apply for an EORI numb
 cds.registration.you-need-a-different-service-iom.vat-registered=VAT-registered
 cds.registration.you-need-a-different-service-iom.not-vat-registered=not VAT-registered
 
+cds.registration.use-a-different-service.title=You need to use a different online service
+cds.registration.use-a-different-service.info=To apply for an EORI number as a public body and not for profit without a UTR you need to use an online form, depending on whether you''re:
```

Comment:
Typo: you''re, Also the sentence looks incomplete to me.

Reply:
Thanks @beckfordp the '' is intentional as it escapes a single '. Its a just a character format thing. If you just have one, then it doesnt show correctly. Also, the colon is on the end as its followed by some separate bullet points. Hope that all makes sense.

Reply:
Yep. Makes sense, plus I've learned something.